cmd/root.go

```diff
@@ -123,6 +114,11 @@ func setupLogger(atmosConfig *schema.AtmosConfiguration) {
 	}
 
 	log.SetOutput(output)
+	if _, err := logger.ParseLogLevel(atmosConfig.Logs.Level); err != nil {
+		//nolint:all
+		log.Fatal("Error parsing log level", "error", err)
+	}
+	log.Debug("Log level set to", "level", log.GetLevel(), "file", atmosConfig.Logs.File)
```

Comment:
```suggestion
	log.Debug("set", "log-level", log.GetLevel(), "log-file", atmosConfig.Logs.File)
```

It turns out that level is an undocumented reserved word. https://github.com/charmbracelet/log/issues/107